deploy-all.yml

```diff
@@ -29,6 +29,15 @@
     - geerlingguy.mysql
     - forward-server-mail
 
+- name: Set up MySQL slave servers
+  hosts: mysql-slave
```

Comment:
Can't we simply add the new group `mysql-slave` to the play we already have?  It seems to be identical.

Reply:
The `mysql_server_id` and `mysql_replication_role` are necessarily different, which is why I put this in a different role. I'm not so great with ansible though, so perhaps there's an easier way to do this and avoid duplication?

Also I've made some other changes to logging and such now, so the roles are more different.